### ISSUE TITLE
CampaignSegmentOpts method: getJsonRepresentation

### ISSUE THREAD

**Issue by cmlanche** [banana-j/bananaj#36]
```
if (getConditions() != null) {
        	JSONArray jsonConditions = new JSONArray();
        	Iterator<AbstractCondition> i = getConditions().iterator();
        	while (i.hasNext()) {
        		jsonConditions.put(i.next());
        	}
        	segmentOpts.put("conditions", jsonConditions);
        }
```

it should be:
```
if (getConditions() != null) {
        	JSONArray jsonConditions = new JSONArray();
        	Iterator<AbstractCondition> i = getConditions().iterator();
        	while (i.hasNext()) {
        		**jsonConditions.put(i.next().getJsonRepresentation());**
        	}
        	segmentOpts.put("conditions", jsonConditions);
        }
```

**Comment by cmlanche** [comment]
my solution is 

```
/**
 * Created by cmlanche on 2019-02-22.
 */
public class CampaignSegmentOptsFix extends CampaignSegmentOpts {

    public CampaignSegmentOptsFix(BuilderFix b) throws ConditionException {
        super(b);
    }

    public JSONObject getJsonRepresentation() {
        JSONObject segmentOpts = new JSONObject();

        if (getSavedSegmentId() != null) {
            segmentOpts.put("saved_segment_id", getSavedSegmentId());
        }

        if (getMatch() != null) {
            segmentOpts.put("match", getMatch().value());
        }

        if (getConditions() != null) {
            JSONArray jsonConditions = new JSONArray();
            Iterator<AbstractCondition> i = getConditions().iterator();
            while (i.hasNext()) {
                jsonConditions.put(i.next().getJsonRepresentation());
            }
            segmentOpts.put("conditions", jsonConditions);
        }

        return segmentOpts;
    }

    public static class BuilderFix extends Builder {
        @Override
        public CampaignSegmentOpts build() {
            try {
                return new CampaignSegmentOptsFix(this);
            } catch (ConditionException e) {
                e.printStackTrace();
            }
            return null;
        }
    }
}
```

usage:
```
```

**Comment by gscriver** [comment]
Fixed in release v0.6.1
